appveyor.yml

```diff
@@ -32,44 +32,53 @@
     - '%AppData%\npm-cache'
 
   environment:
+    # https://github.com/jasongin/nvs/blob/master/doc/CI.md
+    NVS_VERSION: 1.3.0
```

Comment:
If you bump this to `1.4.0` it looks like the rate limit issues should go away https://github.com/jasongin/nvs/releases/tag/v1.4.0